force-app/main/default/lwc/rd2RecurringDonation/rd2RecurringDonation.html

```diff
@@ -210,5 +209,7 @@
     </div>
     <c-update-payment-method-modal is-bank-payment-allowed={allowACHPaymentMethod} current-record={currentRecord} open-update-payment-method={openUpdatePaymentMethod} onclose={handleClose}></c-update-payment-method-modal>
     <c-change-amount-or-frequency-modal is-initially-monthly-donation={isInitiallyMonthlyDonation} is-elevate-donation={isElevateDonation} fixed-installments-label={fixedInstallmentsLabel} day-of-month-field-label={dayOfMonthFieldLabel} default-record-type-id={defaultRecordTypeId} current-record={currentRecord} open-change-amount-or-frequency={openChangeAmountOrFrequency} onclose={handleClose}></c-change-amount-or-frequency-modal>
-    <c-stop-recurring-donation-modal current-record={currentRecord} open-stop-recurring-donation={openStopRecurringDonation} onclose={handleClose}></c-stop-recurring-donation-modal>
+    <template if:true={openStopRecurringDonation}>
+        <c-stop-recurring-donation-modal current-record={currentRecord} open-stop-recurring-donation={openStopRecurringDonation} onclose={handleClose}></c-stop-recurring-donation-modal>
```

Comment:
Ideally, the jest test should be updated to reflect the front end change.

Reply:
Thanks for the suggestion. It has been added